src/wh_crypto.c

```diff
@@ -36,6 +36,7 @@
 #include "wolfssl/wolfcrypt/types.h"
 #include "wolfssl/wolfcrypt/error-crypt.h"
 #include "wolfssl/wolfcrypt/asn.h"
+#include "wolfssl/wolfcrypt/asn_public.h"
```

Comment:
asn.h include asn_public.h.  Were you getting a compile warning/error without asn_public.h explicitly included?

Reply:
I'm dumb, removed